recipe/meta.yaml

```diff
@@ -39,6 +105,9 @@ build:
     - "$RPATH/ld-linux-x86-64.so.2"
     - "$RPATH/libm.so.6"
   skip: true  # [osx]
+{% if major_minor == "10.1" %}
+  skip: true  # [ppc64le]
+{% endif %}
```

Comment:
Apparently 10.1 doesn't work here. Anthony would you be able to share details?

cc @jayfurmanek

Reply:
So, basically, the 10.1 ppc64le package is structured internally slightly differently. For example, 

* Libraries seem to go into `targets/ppc64le-linux/lib` rather than `lib64` normal with other linux64 and later ppc64le package, and
* There seems to be an issue with extracting the cublas headers.

My priority is to get cudatoolkit off of the test label, and since defaults does not have a 10.1 ppc644le package, this should not interfere with any channel priority issues. I am hoping that I made this easy for people to come back to if needed, and just want to skip it for now. I gave it the old college try!

Reply:
Ofc! Just trying to make sure people know what works or doesn't so it can revisited :)

Reply:
Right! It would nice to keep a matrix of these things somewhere.

Reply:
The description you gave is already great! 😄

Filed issue ( https://github.com/conda-forge/cudatoolkit-feedstock/issues/12 ) referencing this.